src/celeritas/inp/Grid.hh

```diff
@@ -55,6 +55,22 @@ struct Grid
     }
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * A uniform grid of increasing, sorted 1D data.
+ */
+struct UniformGrid
+{
+    using VecDbl = std::vector<double>;
+
+    double xmin{};
+    double xmax{};
```

Comment:
Should these also be an `EnumArray<Bounds, real_type> x;`?

Reply:
Good points. I made the change; if it seems better we'll keep it, otherwise I can revert. I also simplified the `CalculatorTestBase`.